pos_require_product_quantity/static/src/js/screens.js

```diff
@@ -16,39 +16,28 @@ odoo.define('pos_require_product_quantity.screens', function (require) {
             var self = this;
             this._super();
 
-            this.$('.pay').click(function () {
-
-                if (self.pos.config.require_product_quantity) {
-
-                    var orderlines = self.pos.get_order().orderlines;
-                    var qty_unset_list = [];
-
-                    for (var i = 0; i < orderlines.length; i++) {
-                        var line = orderlines.models[i];
-                        if (line.quantity === 0) {
-                            qty_unset_list.push(line);
-                        }
-                    }
-                    if (qty_unset_list.length > 0) {
+            if (self.pos.config.require_product_quantity) {
+                this.$('.pay').click(function () {
+                    var lines_without_qty = _.filter(
```

Comment:
I'm a js n00b, what is the value of `_` here?

Reply:
Just to simplify the code by using [underscore](https://underscorejs.org/) utility methods ☺️

In this case it resulted in cleaner code than a `for .. loop`

Reply:
Ha ok, it's a library, I thought it was some kind of js magic.

Beautiful, _kill all the loops_!